support-frontend/app/controllers/Application.scala

```diff
@@ -201,7 +201,7 @@ class Application(
       geoData = geoData,
       shareImageUrl = shareImageUrl(settings),
       shareUrl = "https://support.theguardian.com/contribute",
-      v2recaptchaConfigPublicKey = recaptchaConfigProvider.v2PublicKey,
+      v2recaptchaConfigPublicKey = recaptchaConfigProvider.get().v2PublicKey,
```

Comment:
So we are not using the test key for the contributions page? So will that still avoid the recaptcha in the same way as the subs pages used to? I _think_ this is probably ok because hopefully anything that breaks the recaptcha would be picked up by the subs tests.

Reply:
it also depends whether contributions team find it annoying when testing!

Reply:
That's right - there is no change to the reCAPTCHA post-deploy tests behaviour on the Contributions Stripe form. The post-deploy tests on the Contributions landing page continue to bypass it using a similar mechanism to the one the Subs checkouts used to have. The only minor difference is that on the Contributions page, the post-deploy tests explicitly check for a valid `_test_username` cookie in addition to the `_post_deploy_user` cookie, and ignore the reCAPTCHA verification result if both cookies are present.

Yep, this was my logic too - as the purpose of this PR is to get the post-deploy tests to pick up a broken reCAPTCHA, the Subs checkouts should be enough.